less/index.less

```diff
@@ -1,59 +1,223 @@
-footer, header, hgroup, menu, nav, section {
-	display: block;
-}
-body {
-	line-height: 1;
-}
-ol, ul {
-	list-style: none;
-}
-blockquote, q {
-	quotes: none;
-}
-blockquote:before, blockquote:after,
-q:before, q:after {
-	content: '';
-	content: none;
-}
-table {
-	border-collapse: collapse;
-	border-spacing: 0;
+// Mixins
+.center() {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	.mb20;
+  }
+  
+  .mb20() {
+	margin-bottom: 20px;
+  }
+  
+  .box-shadow (@x: 0px, @y: 3px, @blur: 5px, @alpha: 0.5) {
+	-webkit-box-shadow: @x @y @blur rgba(0, 0, 0, @alpha);
+	-moz-box-shadow: @x @y @blur rgba(0, 0, 0, @alpha);
+	box-shadow: @x @y @blur rgba(0, 0, 0, @alpha);
 }
+  
+  // Variables
+  @main-color: @plain-old-red;
+  @plain-old-red: rgb(189, 167, 167);
+  @zodiac-blue: #112546;
+  @pacifico: 'Pacifico', cursive;
+  @economica: 'Economica', sans-serif;
+  @Sans: 'Julius Sans One', sans-serif;
+  @candal: 'Candal', sans-serif;
+  @background: lightgrey;
+  @emperor: #545454;
+  @primary: #B5BBC5;
+  @top-padding: 20px;
+  @bottom-padding: 10px;
+  
+  html {
+	box-sizing: border-box;
+	font-size: 62.5%; 
+		.box-shadow(5px, 5px, 6px, 0.3);
+  }
+  
+  *, *:before, *:after {
+	box-sizing: inherit;
+  }
+  
+  h1, h2, h3, h4, h5 {
+	font-size: 5rem;
+	color: @emperor;
+	font-family: @candal;
+
+  }
+  
+  img {
+   width: 100%;
+   height: auto;
+	.box-shadow(5px, 5px, 6px, 0.3);
+} 
+  
+  p {
+	line-height: 1.25;
+	margin: 20px 0;
+	font-size: 1.8rem;
+	color: @zodiac-blue;
+	padding: @top-padding;
+	font-family: @candal;
+  }
+
+  li {
+	  font-family: @sans;
+	  font-size: 4rem;
+	  color: @zodiac-blue;
+  }
+  
+  strong {
+	font-weight: bold;
+	color: @zodiac-blue;
+	// color: red;
+  }
+  
+  .container {
+	max-width: 800px;
+	margin: 0 auto;
+	font-size: 1.6rem;
+	// background-color: @background;
+	border: 4rem solid red;
+  }
+  
+  header {
+	border-bottom: 1px solid black;
+	margin: 20px 0;
+	padding: 20px 0;
+	display: flex;
+	justify-content: space-between;
+	display: flex;
+	// color: @emperor;
+  
+	@media( max-width: 500px ) {
+	  flex-direction: column;
+	  align-items: center;
+	}
+	
+	h1 {
+	  font-size: 5rem;
+	  font-family: @candal;
+	  
+	  @media( max-width: 500px ) {
+		margin-bottom: 20px;
+	  }
+	}
+
+	h3 {
+		font-size: 2rem;
+	}
+	
+	nav {
+	  margin-left: 62px;
+	//   background-color: @emperor;
+	  height: 5rem;
+	  text-align: center;
+
+	  @media( max-width: 500px ) {
+		margin-left: 0;
+	  }
+	  
+	  a {
+		color: @primary;
+		margin-left: 20px;
+		font-size: 1.8rem;
+		padding: 15px;
+	  }
+	  
+	  .first-anchor {
+		margin-left: 0;
+	  }
+	}// nav
+  } // header
+
+  .main-content {
+	border-top: 2px solid black;
+	border-bottom: 2px solid black;
+	margin: 75px 0 35px 0;
+	padding: 35px 0;
+	font-size: 16px;
+	line-height: 18px;
+	width: 100%;
+	display: flex;
+	flex-wrap: wrap;
+ 
+ .main-content .top-content {
+	 display: flex;
+ }
+  
+//  .main-content {
```

Comment:
still enjoying your thought process when you decide to change code and leave the old code commented out